sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2692,20 +2692,15 @@ pi_result hip_piQueueFlush(pi_queue command_queue) {
 ///
 /// \return PI_SUCCESS
 pi_result hip_piextQueueGetNativeHandle(pi_queue queue,
-                                        pi_native_handle *nativeHandle) {
+                                        pi_native_handle *nativeHandle,
+                                        int32_t *NativeHandleDesc) {
+  (void)NativeHandleDesc;
```

Comment:
I think it is worth initializing desc with 0

Reply:
OK, done.